tools-local/unix/acquire-tool.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+
+# Downloads the declared version of the specified tool.
+
+usage()
+{
+    echo "Usage: $0 ToolName"
+    echo "  ToolName: Name of the tool to download."
+    echo ""
+    echo "Downloads the declared version of the specified tool from the corresponding URL specified in the .toolversions file."
+    echo "If download succeeds then, returns the path to the executable."
+    echo "Exit 1 if download fails."
+}
+
+if [ $# -lt 1 ]; then
+    usage
+    exit 1
+fi
+
+if [ -z "$1" ]; then
+    echo "Argument passed as ToolName is empty. Please provide a non-empty string."
+    exit 1
+fi
+
+toolName="$1"
+scriptPath="$(cd "$(dirname "$0")"; pwd -P)"
+. "$scriptPath/tool-helper.sh"
+declaredVersion="$(get_tool_config_value "$toolName" "DeclaredVersion")"
+
+
+# Downloads the package corresponding to the tool, and extracts the package.
+download_extract()
+{
+    # Get the download URL
+    downloadUrl="$(get_tool_config_value "$toolName" "DownloadUrl")"
+
+    if [ $? -ne 0 ]; then
+        echo "$downloadUrl"
+        exit 1
+    fi
+
+    downloadPackageName=$(get_download_package_name "$toolName")
+
+    if [ $? -ne 0 ]; then
+        echo "$downloadPackageName"
+        exit 1
+    fi
+
+    downloadUrl="$downloadUrl$downloadPackageName"
+
+    # Create folder to save the downloaded package, and extract the package contents.
+    repoTools=$(get_repository_tools_downloads_folder "$toolName")
```

Comment:
Good catch. Updated `get_repository_tools_downloads_folder` to return the path to the specified folder under Tools/downloads.